resolwe_bio/tools/append_sample_info.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+"""Extract reference (FASTA) and sample names from the VCF file."""
+
+import os
+import argparse
+
+from pysam import VariantFile
+from resolwe_runtime_utils import error, warning
+
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument('vcf_file', help="VCF file (can be compressed using gzip/bgzip).")
+parser.add_argument('summary', help="Summary file to append to.")
+args = parser.parse_args()
+
+try:
+    vcf = VariantFile(args.vcf_file)
+except (OSError, ValueError):
+    print(error('Input VCF file does not exist or could not be correctly opened.'))
+    exit(1)
+
+vcf_header = vcf.header
+header_records = {record.key: record.value for record in vcf_header.records}
+
+with open(args.summary, "a") as out_file:
+    try:
+        fasta_name = os.path.basename(header_records['reference'])
+    except KeyError:
+        fasta_name = ''
+        print(warning('Reference sequence (FASTA) name could not be recognized from the VCF header.'))
```

Comment:
For warnings, I would use [warnings](https://docs.python.org/3/library/warnings.html#module-warnings).

```
import warnings
# ...
warn_msg = 'Reference sequence (FASTA) name could not be recognized from the VCF header.'
print(warning(warn_msg))
warnings.warn(warn_msg)
```

Reply:
We want to use the `resolwe_runtime_utils` process error/warning/info system for the resolwe process support scripts. As a standalone script outside of resolwe universe, the suggested approach would make sense.